mne/preprocessing/ica.py

```diff
@@ -938,7 +938,8 @@ def find_bads_ecg(self, inst, ch_name=None, threshold=None,
             if threshold is None:
                 threshold = 0.25
             if isinstance(inst, _BaseRaw):
-                sources = self.get_sources(create_ecg_epochs(inst)).get_data()
+                sources = self.get_sources(create_ecg_epochs(
+                    inst, ch_name, keep_ecg=True)).get_data()
```

Comment:
No, I don't think so - note we don't return the created sources anyways. And there is no kwarg on the input side.